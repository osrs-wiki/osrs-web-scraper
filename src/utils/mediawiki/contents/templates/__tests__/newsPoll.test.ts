import NewsPollTemplate from "../newsPoll";

describe("NewsPollTemplate", () => {
  test("it should render with a number and question", () => {
    expect(new NewsPollTemplate("test?", 5).build()).toMatchSnapshot();
  });

  test("it should render with only a question", () => {
    expect(new NewsPollTemplate("test?").build()).toMatchSnapshot();
  });
});
