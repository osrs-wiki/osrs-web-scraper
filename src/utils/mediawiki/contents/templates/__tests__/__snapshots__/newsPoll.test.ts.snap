// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsPollTemplate it should render with a number and question 1`] = `
MediaWikiTemplate {
  "children": undefined,
  "name": "News Poll",
  "params": Array [
    Object {
      "key": "",
      "value": "5",
    },
    Object {
      "key": "",
      "value": "test?",
    },
  ],
}
`;

exports[`NewsPollTemplate it should render with only a question 1`] = `
MediaWikiTemplate {
  "children": undefined,
  "name": "News Poll",
  "params": Array [
    Object {
      "key": "qtext",
      "value": "test?",
    },
  ],
}
`;
